custom/opm/beneficiary.py

```diff
@@ -635,20 +635,9 @@ def forms(self):
     @property
     def all_conditions_met(self):
         if self.status == 'mother':
-            relevant_conditions = [
-                self.child_attended_vhnd,
-                self.child_growth_calculated,
-                self.child_received_ors,
-                self.child_condition_four,
-                self.child_breastfed,
-            ]
```

Comment:
This condition was not checking the same stuff as child_followup:
```python
self.child_attended_vhnd,
self.child_received_ors,
self.child_growth_calculated,
self.child_weighed_once,
self.child_birth_registered,
self.child_received_measles_vaccine,
self.child_breastfed,
```
The CMR just checked to see if all conditions were met, whereas the BPR checks 3 categories of conditions depending on the term, THEN combines them to get the total (that's why they have different methods).  I've removed the duplication in condition specification.